arches/app/templates/views/components/widgets/datepicker.htm

```diff
@@ -87,16 +89,19 @@
 </div>
 <div>
     <div class="form-group col-xs-12 pad-no crud-widget-container">
-        <div style="pad-hor">
+        <div class="pad-hor">
             <div class="input-group date">
-                <input type="text" data-bind="value: defaultValue, placeholder: placeholder, datepicker: {format: dateFormat, viewMode: viewMode, maxDate: maxDate, minDate: minDate}" class="form-control input-md" style="height: 32px;"><span class="input-group-addon"><i class="fa fa-calendar fa-lg"></i></span>
+                <input type="text" data-bind="value: defaultValue, placeholder: placeholder, attr: {'aria-label': placeholder},
```

Comment:
the aria-label value should be "$root.translations.defaultValue" instead of "placeholder"